### PR TITLE
Redesigned Index Page to look more modern and clean

### PR DESCRIPTION
**Pull request by nicosammito**
Bildschirmfoto 2022-06-22 um 00 22 10

### CODE REVIEW

**Comment by froxcey** [comment]
It's a pretty cool concept. Two problems:
I find everything pretty jammed together, so when I visit it kinda feels restricted, which is totally bad for a page for OS. You might want to space things out a bit more.
You have removed the Under Development notice from every page, which is not ideal since it is meant to be on every single page so when someone doesn't go to home page (for example searches on Google and directly click on Download link), they know that the OS is still under development.

**Comment by nicosammito** [comment]
> It's a pretty cool concept. Two problems:
> I find everything pretty jammed together, so when I visit it kinda feels restricted, which is totally bad for a page for OS. You might want to space things out a bit more.
> You have removed the Under Development notice from every page, which is not ideal since it is meant to be on every single page so when someone doesn't go to home page (for example searches on Google and directly click on Download link), they know that the OS is still under development.

oh yeh I will add the alert thing on every page, forgot to add this 😄. Hmm yeh its pretty bunched up because of the backdrop filter effect, but maybe I kind find a solution for this

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,75 +1,88 @@
...
+                  -webkit-background-clip: text;
+                  -webkit-text-fill-color: transparent;
+                }
+      
+                .video__player {
+                    width: 100vw;
+                    position:relative;
+                    margin-bottom: ${useMobileBar ? "-20vh" : "-60vh"};
+                }
+
```
Comment (froxcey) [inline src/pages/index.tsx:72]: it's okay to group related style definition with spaces, but this is unnecessary spacing. You might want to review your css definition and properly group definitions.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,75 +1,88 @@
...
-          padding: 0;
-          border-top: none !important;
-        }
-        .header {
-          font-size: 72px;
-          background: -webkit-linear-gradient(#eee, ${theme.palette.accents_2});
-          -webkit-background-clip: text;
-          -webkit-text-fill-color: transparent;
-        }
+    const theme = useTheme();
```
Comment (froxcey) [inline src/pages/index.tsx:9]: Please use two spaces as spacing, not four or tabs. This is tsx convention.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,75 +1,88 @@
-import { Button, Card, Text, Spacer, useTheme } from '@geist-ui/core';
+import {Button, Text, Spacer, useTheme} from '@geist-ui/core';
 import NextLink from 'next/link';
 import dynamicWidth from '@/lib/dynamic-width';
-import { useState } from 'react';
+import {useState} from 'react';
```
Comment (froxcey) [inline src/pages/index.tsx:4]: Please add space after { and before }
Also, you might want to install a linter.

**Review by froxcey — COMMENTED** [review]
I reviewed your code, and there are several code quality issues. You might want to install a formatter/linter like prettier.

1. Use 2 spaces for spacing
2. Use `-` to indicate a word and `__` to indicate child
3. Spaces around curly braces

I see that you added the alert back, but the width is not 100%. You might want to make it 100% width, or something like that but with a bit of spacing. You get the idea.

**Review by froxcey — CHANGES_REQUESTED** [review]
Thank you for the changes

You might want to review the light theme, and I'm also going to do a bit of editing if that's okay.

**Comment by froxcey** [comment]
I made some edit to 
- Increase the spacing
- Revert the alert to the top of the screen
- Make repo list more concise
- Change transparent background to 0.3 opacity (better visibility)

Screenshot 2022-06-22 at 10 18 13 AM
Screenshot 2022-06-22 at 10 20 36 AM
Screenshot 2022-06-22 at 10 20 50 AM

If it is good, I will commit these changes

**Comment by nicosammito** [comment]
> I made some edit to
> 
> * The spacing to have more room and better adapt to all screen sizes
> * Revert the alert to the top of the screen
> * Make repo list more concise
> * Change transparent background to 0.3 opacity (better visibility)
> 
> Screenshot 2022-06-22 at 10 18 13 AM
> 
> Screenshot 2022-06-22 at 10 20 36 AM
> 
> Screenshot 2022-06-22 at 10 20 50 AM
> 
> If it is good, I will commit these changes

yeh kind of like that, but the spacing between the heading and the 2 buttons is way too big in my opinion. I would decrease this by half.

**Comment by froxcey** [comment]
Screenshot 2022-06-22 at 10 26 07 AM

This is what it looks like to decrease the spacing by half. Are you sure? This will make everything crowded together in the middle of the screen. Also, we might also need other member's opinion.

**Comment by nicosammito** [comment]
> Screenshot 2022-06-22 at 10 26 07 AM
> 
> This is what it looks like to decrease the spacing by half. Are you sure? This will make everything crowded together in the middle of the screen. Also, we might also need other member's opinion.

yes absolutely, its way better for the UX because we're grouping things together that belong together. Later we could also add like a little paragraph text between the buttons and the heading.

Bildschirmfoto 2022-06-22 um 04 31 56

Here is an example by apple: They group components together to achieve a better UX

**Comment by nicosammito** [comment]
And also don't forget the user looks instinctively in the middle of the screen, so we should group the important things together, so the user can easily reach what he wants to do

**Comment by froxcey** [comment]
Well I just ask the team, and they prefer smaller spacing. I will commit now. Since this is a lot of changes, I might need someone else to also review or get an extra confirmation

**Review by froxcey — APPROVED** [review]
LGTM, just need a confirmation from web dev lead.

**Comment by nicosammito** [comment]
> LGTM, just need a confirmation from web dev lead.

**Comment by nicosammito** [comment]
UPS

File: src/styles/components/cards.module.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  border: 1px solid rgba(0, 0, 0, 0.1) !important;
+}
+
+.card__white:hover {
+  border: 1px solid rgba(0, 0, 0, 0.3) !important;
+}
+
+.card__dark {
+  background: rgba(0, 0, 0, 0.3) !important;
+}
```
Comment (nicosammito) [inline src/styles/components/cards.module.css:17]: different design on dark mode? little bit confusing. And also the border can't be seen.
```suggestion
.card {
  backdrop-filter: blur(2rem);
  -webkit-backdrop-filter: blur(2rem);
}

.card__white {
  background: rgba(255, 255, 255, 0.2) !important;
  border: 1px solid rgba(0, 0, 0, 0.3) !important;
}

.card__white:hover {
  border: 1px solid rgba(0, 0, 0, 0.4) !important;
}

.card__dark {
  background: rgba(0, 0, 0, 0.2) !important;
  border: 1px solid rgba(255, 255, 255, 0.3) !important;
}

.card__dark:hover {
  border: 1px solid rgba(255, 255, 255, 0.4) !important;
}
```

Comment (froxcey) [inline src/styles/components/cards.module.css:17]: Well, dark mode already have a really nice border, so there's no need to change it. As for the light mode, yes it does need some enhancement.

Comment (froxcey) [reply src/styles/components/cards.module.css:17]: Well, dark mode already have a really nice border, so there's no need to change it. As for the light mode, yes it does need some enhancement.

Comment (nicosammito) [reply src/styles/components/cards.module.css]: hmm yeh I kind of understand that but in my opinion the website should look similar to the OS itself and the default borders are not like the os design. Also for consistency reasons we should have the same design for both white and dark mode, shouldn't we?

**Review by AZProductions — APPROVED** [review]
Before I merge it, this is the biggest pr we've ever had, this is awesome! Everyone including Froxcey and me are extremely happy with this change. I saw your discord message, accepting your contributor role right now.
- Pigpot

**Pull request merged**